package.json

```diff
@@ -23,10 +23,10 @@
     "@angular/platform-browser-dynamic": "18.2.4",
     "@angular/router": "18.2.4",
     "@fontsource/roboto": "^4.5.8",
-    "@ng-doc/app": "18.0.3",
-    "@ng-doc/builder": "18.0.3",
-    "@ng-doc/core": "18.0.3",
-    "@ng-doc/ui-kit": "18.0.3",
+    "@ng-doc/app": "^17.6.16",
+    "@ng-doc/builder": "^17.6.16",
+    "@ng-doc/core": "^17.6.16",
+    "@ng-doc/ui-kit": "^17.6.16",
```

Comment:
Is it this one? https://github.com/advisories/GHSA-pfq8-rq6v-vf5m

Reply:
Yes, @semicolin we need to downgrade because of a critical vulnerability in the latest version.
![image](https://github.com/user-attachments/assets/0dd0ac66-3815-441d-a015-23c6e15f5639)